sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/RequestResponseChannel.java

```diff
@@ -102,24 +102,19 @@
         this.replyTo = entityPath.replace("$", "") + "-client-reply-to";
         this.messageSerializer = messageSerializer;
         this.sendLink = session.sender(linkName + ":sender");
-        final Target target = new Target();
-        target.setAddress(entityPath);
-        this.sendLink.setTarget(target);
-        sendLink.setSource(new Source());
-        this.sendLink.setSenderSettleMode(senderSettleMode);
```

Comment:
Yes. They are set in protected methods below because ServiceBus's RequestResponse channel is set up differently. So I am expecting that SB will implement those overrides.

Reply:
I can pass these later. My original design was that these could be overridden in an extending class. I'll revert this.

Reply:
That design is good. Thanks for letting me know.